src/wazuh_db/wdb_agents.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Wazuh DB helper module for agents database
+ * Copyright (C) 2015-202, Wazuh Inc.
```

Comment:
There is a number missing: `(C) 2015-202`

Reply:
Fixed